app/src/main/res/values-de/strings.xml

```diff
@@ -159,8 +159,6 @@
     <string name="nc_make_call_private">Unterhaltung privat machen</string>
     <string name="nc_delete_call">Unterhaltung löschen</string>
     <string name="nc_delete">Löschen</string>
-    <string name="nc_delete_conversation_default">Bitte bestätigen Sie, dass Sie diese Unterhaltung entfernen möchten.</string>
```

Comment:
those strings are still used in 11 😕

Reply:
Don't know really, linter warns about a couple of unused string though we might remove them in the near future (for v12 then)

Reply:
I removed them in https://github.com/nextcloud/talk-android/pull/1218/files#diff-2f516f16cffd61f534bc36fffec911d1b2909323ee3717ddbeec5db40dd4264cL138 which is not going to be backported

Reply:
rebased and fixed